### PR TITLE
Improve the `awful` example test tempate

### PR DESCRIPTION
**Pull request by Elv13**

### CODE REVIEW

File: tests/examples/awful/template.lua [hunk 1]
```diff
@@ -5,9 +5,11 @@ local cairo      = require("lgi").cairo
...
 local pangocairo = require("lgi").PangoCairo
 
-local color     = require( "gears.color" )
-local shape     = require( "gears.shape" )
-local beautiful = require( "beautiful"   )
+local color     = require( "gears.color"    )
+local shape     = require( "gears.shape"    )
+local beautiful = require( "beautiful"      )
+local wibox     = require( "wibox"          )
+local titlebar  = require( "awful.titlebar" )
```
Comment (blueyed) [inline tests/examples/awful/template.lua]: Unused `titlebar`.

File: tests/examples/awful/template.lua [hunk 2]
```diff
@@ -20,16 +22,6 @@ local cr  = cairo.Context(img)
 local pango_crx = pangocairo.font_map_get_default():create_context()
 local pl = pango.Layout.new(pango_crx)
```
Comment (blueyed) [inline tests/examples/awful/template.lua]: Unused `pl` (+ `pango_crx`).

Comment (Elv13) [inline tests/examples/awful/template.lua]: Whoops, the fixup from the original PR was merged into the wrong commit...

**Comment by blueyed** [comment]
@Elv13 
Please fix.

**Comment by Elv13** [comment]
I am traveling right now. It's gonna take a while. Sorry about it (the same is true for the other PRs)

**Pull request merged**